scripts/build.ts

```diff
@@ -19,7 +19,7 @@ function buildPackage() {
   const packageDir = new URL("./packages/web-features/", rootDir);
   const filesToCopy = ["LICENSE.txt", "types.ts"];
 
-  const json = stringify(features);
+  const json = stringify(data);
```

Comment:
Ugh, sorry, I only just now noticed this. If we're going to the trouble of having named exports, we ought to generate three JSON files, then change `packages/web-features/package.json` to have something like:

```
  "exports": {
    ".": "./index.js",
    "./features.json": "./features.json",
    "./groups.json": "./groups.json",
    "./snapshots.json": "./snapshots.json",
  },
```

(and soon `"./browsers.json": "./browsers.json"`)

Reply:
I can do that, but what's the effect for consumers of the package? Does it change how you can import things?

Also, do you want a single JSON file or multiple ones for GitHub releases?

Reply:
I think just one JSON is fine. I'm expecting we'll eventually have three release artifacts: npm package, JSON, and full-sized expanded JSON.

Reply:
Great, one JSON file is what I'd like too :)

Reply:
@foolip and I chatted about this a bit. This could be a YAGNI thing, so let's not for now.